Android.mk

```diff
@@ -116,10 +119,24 @@ LOCAL_C_INCLUDES                                         := \
     $(NULL)
 
 LOCAL_CFLAGS                                                                := \
-    $(OPENTHREAD_COMMON_FLAGS)                                                 \
+    $(OPENTHREAD_PUBLIC_CFLAGS)                                                \
+    $(OPENTHREAD_PRIVATE_CFLAGS)                                               \
     $(OPENTHREAD_PROJECT_CFLAGS)                                               \
     $(NULL)
 
+LOCAL_EXPORT_CFLAGS                                        := \
+    -DOPENTHREAD_CONFIG_FILE=\<openthread-config-android.h\>  \
+    $(OPENTHREAD_PUBLIC_CFLAGS)                               \
+    $(OPENTHREAD_PROJECT_CFLAGS)                              \
+    $(NULL)
+
+LOCAL_EXPORT_C_INCLUDE_DIRS     := \
+    $(OPENTHREAD_PROJECT_INCLUDES) \
+    $(LOCAL_PATH)/include          \
+    $(LOCAL_PATH)/src              \
+    $(LOCAL_PATH)/src/core         \
```

Comment:
is core headers necessary to be exported?

Reply:
Yes, we need to export `openthread-core-config.h` so that users can check which features are supported. This header is requires core headers.